crates/ide_assists/src/handlers/generate_function.rs

```diff
@@ -503,12 +507,25 @@ fn deduplicate_arg_names(arg_names: &mut Vec<String>) {
     }
 }
 
-fn fn_arg_name(arg_expr: &ast::Expr) -> String {
+fn fn_arg_name(ctx: &AssistContext, arg_expr: &ast::Expr) -> String {
```

Comment:
```suggestion
fn fn_arg_name(sema: &Semantics<RootDatabase>, arg_expr: &ast::Expr) -> String {
```
No need to pass `AssistContext` here if we only need the semantics